uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/Service/LeaveRequestCalendarFeedData.php

```diff
@@ -87,6 +94,21 @@ public function get() {
     return $leaveData;
   }
 
+  /**
+   * Returns the Leave type label from the leave type ID. For cases
+   * where the typeId is not found in the absence types list, i.e for
+   * leave types with their hide_label property as true, the leave label
+   * is returned as 'Leave'.
```

Comment:
I'm not sure if the `hide_label` makes any difference here, as the `getEnabledLeaveTypes()` only return a list of enabled absence types, regardless if the label should be hidden or not. I'm missing something?

Reply:
Yeah, thats true.